packages/dashboard/src/views/TestDetailsView.tsx

```diff
@@ -9,6 +9,7 @@ export function TestDetailsView(): React.ReactNode {
 
   const { loading, error, data } = useGetInstanceQuery({
     variables: { instanceId },
+    pollInterval: 1500,
```

Comment:
This might be too short for some situations. For my situation its awesome. This could go away if everything was changes to websockets. but that might be a fat task. Maybe we consider this being an env var or something in the future?

Reply:
Let's see if people complain :)